languages/payplug-en_US.po

```diff
@@ -14,6 +14,9 @@ msgstr ""
 "Content-Transfer-Encoding: 8bit\n"
 "X-Generator: Poedit 3.0.1\n"
 
+msgid "Advanced Settings"
```

Comment:
"Advanced Settings" better use snake case for translations ids

Reply:
Done